src/coreclr/vm/peimagelayout.cpp

```diff
@@ -13,6 +13,13 @@
 #include "amsi.h"
 #endif
 
+#if defined(CORECLR_EMBEDDED)
+extern "C"
+{
+#include "../../../libraries/Native/AnyOS/zlib/pal_zlib.h"
```

Comment:
I don't like long relative paths in general, they are a burden when structure of the source tree changes. It seems it would be nicer to add the ../../../libraries/Native/AnyOS/zlib to the include paths in cmake scripts and have just
```c++
#include "pal_zlib.h"
```
here. It would be also nice to define the libraries path at one place in the cmake files and use it as a basis for this and the path in src/coreclr/vm/CMakeLists.txt.

Reply:
There was another long `pall_zlib.h` include in apphost/static. I have changed that too.